meta-lxatac-bsp/recipes-bsp/barebox/files/patches/0103-ARM-stm32mp-add-Linux-Automation-TAC-Generation-3.patch

```diff
@@ -707,27 +698,7 @@ index fc3a2386dbb9..0afbb1a1fa4f 100644
  &gpiof {
  	gpio-line-names = "TP36", "TP37", "", "", "OLED_CS", /*  0 */
  	"", "", "", "", "",                                  /*  5 */
-@@ -409,7 +333,7 @@ &sdmmc2 {
- &spi2 {
- 	pinctrl-names = "default";
- 	pinctrl-0 = <&spi2_pins_c>;
--	cs-gpios = <&gpiof 12 GPIO_ACTIVE_LOW>;
-+	cs-gpios = <&gpiof 12 (GPIO_ACTIVE_LOW | GPIO_OPEN_DRAIN)>;
- 	status = "okay";
- };
- 
-@@ -471,6 +395,10 @@ switch: switch@0 {
- 		interrupt-parent = <&gpioa>;
- 		interrupts = <6 IRQ_TYPE_EDGE_RISING>;
- 
-+		/* Reduce RGMII EMI emissions by reducing drive strength */
```

Comment:
Why is this being removed?

Reply:
This change has made it into the mainline Linux https://github.com/torvalds/linux/commit/7fd195f01ae52871b04b752447f865e1a6661487 and mainline barebox https://github.com/barebox/barebox/commit/a0991ad989b2b678aecf1ba85ee6daad469aff89 .